### PR TITLE
Switch to OpenAIP google cloud storage

### PR DESCRIPTION
**Pull request by Marilyth**
OpenAIP now has a storage bucket through which airspace files can be downloaded.
The google cloud storage library and the OpenAIP API require authentication, so the google cloud storage API is used instead.

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
I have a problem to use the downloaded data, see

download_and_use_problem

also clicking the Update/Download button downloads files and then by apply I get the error message

```
ERROR: Tried to draw airspaces without asp files.
INFO: Downloading to /home/reimar/.config/mss/airports.csv. This might take a while.
INFO: 3152KB Downloaded
INFO: 6542KB Downloaded
INFO: 9823KB Downloaded
ERROR: Tried to draw airspaces without asp files
```

**Comment by Marilyth** [comment]
You selected airports and airspaces to be drawn, but selected none to be drawn in the combobox. Which caused this error.
I suppose it would make sense to disable the checkbox in those cases.

**Comment by Marilyth** [comment]
The button/checkbox now gets enabled/disabled by itself, depending on the current situation with the combobox.
This should prevent the user from applying empty airports or airspaces to begin with.

File: mslib/utils/airdata.py [hunk 1]
```diff
@@ -145,7 +141,7 @@ def update_airspace(force_download=False, countries=["de"]):
         if (force_download or is_outdated or not file_exists) \
                 and QtWidgets.QMessageBox.question(
                     None, "Allow download",
-                    f"The selected {country} airspace needs to be downloaded ({data[-1]})"
+                    f"The selected {country} airspace needs to be downloaded ({round(int(data[-1]) / (1024**2), 3)}MB)"
```
Comment (ReimarBauer) [inline mslib/utils/airdata.py]: have a look on this ;) 
```
import humanfriendly
humanfriendly.format_size(NUMBER)
```
more examples https://pypi.org/project/humanfriendly/#description

we have this in our stack.

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
a small change for using humanfriendly.

File: mslib/utils/airdata.py (hunk 1 above)
Comment (Marilyth) [reply mslib/utils/airdata.py]: Done

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**